docs/release/old-release-removal.md

```diff
@@ -82,3 +83,23 @@ If we really need RPMs from a removed release series, we can look at the text
 files in `/p/vdt/public/html/release-info/attic` to determine the exact NVRs we
 need, and download them from Koji.
 
+
+
+Template Email
+--------------
+
+>   On <DAYNAME, MONTH DAY>, the OSG will be removing the OSG <3.X> release
+>   series from our repositories.  This includes both RPMs and tarballs hosted
+>   on repo.grid.iu.edu.
```

Comment:
Use repo.opensciencegrid.org instead

Reply:
When did that come into existence?

Should we update the yum repo files in osg-release to point to that?

Reply:
It's been up for quite a long time now -- we probably should have fixed all of that in the transition from software -> repo. I'm thinking about making a JIRA epic for it so you can standby